donut/modules/groups/helpers.py

```diff
@@ -157,6 +157,44 @@ def get_position_data(fields=None):
     return user_position_arr
 
 
+def add_position(group_id, pos_name):
+    ''' 
+    Inserts new position into the database associated
+    with the given group and with the given name
+
+    Arguments: 
+        group_id: the id of the group you want to insert the position into
+        pos_name: name of the position to be created
+    '''
+    # Construct the statement
+    s = "INSERT INTO positions (group_id, pos_name) VALUES (%d, '%s')"
+    s = s % (group_id, pos_name)
```

Comment:
Would be better to let `pymysql` do the query substitution to avoid SQL injection attacks